linux/system/selinux.sls

```diff
@@ -1,30 +1,28 @@
 {%- from "linux/map.jinja" import system with context %}
-{%- if system.enabled %}
+{%- if system.selinux is defined %}
```

Comment:
IMO you should keep if `system.enabled` here or extend it to `if system.enabled and system.selinux is defined`.